### ISSUE TITLE
[4.8 beta] Issue on profile dropdown with checked adaptive layer

### ISSUE THREAD

**Issue by Liger0** [Ultimaker/Cura#8651]
**Application version**
4.8 beta

**Platform**
Windows 10 x64

**Printer**
Cr10S

**Reproduction steps**
1. Select Dynamic profile
2. view, uncheck, check the adaptive layer option

**Screenshot(s)**
https://youtu.be/CRPbopEmrkU

**Actual results**
The options are not visible unless you disable and re-enable the feature.

**Expected results**
Visible options at least with enabled feature

**Project file**
(For slicing bugs, provide a project which clearly shows the bug, by going to File->Save Project. For big files you may need to use WeTransfer or similar file sharing sites. G-code files are not project files!)

**Log file**
(See https://github.com/Ultimaker/Cura#logging-issues to find the log file to upload, or copy a relevant snippet from it.)

**Additional information**
(Extra information relevant to the issue.)

**Comment by Ghostkeeper** [comment]
Indeed, this reproduces. And afterwards, you can press the reset button too, which makes the settings go hide again.

**Comment by GregValiant** [comment]
Is this still an issue in current versions of Cura (5.8.0 and up)? Can this be closed?
